src/utils/auth/signUp.js

```diff
@@ -0,0 +1,15 @@
+import { SIGN_UP_URL } from '../../URLs';
+import makeRequest from '../services/makeRequest';
+
+export const signUp = (email, password) => {
```

Comment:
There are a couple of files that have both `export` and `export default` for its sole exported thing.